config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml

```diff
@@ -391,3 +391,46 @@ periodics:
     testgrid-tab-name: aks-engine-azure-windows-1903-master
     testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
     description: Runs SIG-Windows release tests on K8s clusters provided by aks-engine (https://github.com/Azure/aks-engine) on Azure cloud"
+- interval: 10h
+  name: ci-kubernetes-e2e-aks-engine-azure-master-windows-1909
+  labels:
+    preset-service-account: "true"
+    preset-azure-aksengine: "true"
+    preset-k8s-ssh: "true"
+    preset-dind-enabled: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20191231-77c7890-master
+      args:
+      - "--job=$(JOB_NAME)"
+      - "--root=/go/src"
+      - "--repo=k8s.io/kubernetes=master"
+      - "--upload=gs://kubernetes-jenkins/logs/"
+      - "--timeout=650"
+      - "--scenario=kubernetes_e2e"
+      - --
+      - "--test=true"
+      - "--up=true"
+      - "--down=true"
+      - "--deployment=aksengine"
+      - "--provider=skeleton"
+      - "--build=host-go"
```

Comment:
`--build=quick` should be used when `--aksengine-deploy-custom-k8s` is enabled.

Reply:
done